pkg/version/version.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // DO NOT EDIT
 // This var is updated automatically as part of the build process
-var Version = "latest"
+var Version = "undefined"
```

Comment:
What about naming this something a bit more explicit, like `ConduitVersionToInject`? Especially bacause AFAICT this version might be different from the client's own version.

Reply:
This version isn't specific to inject -- it's the global version used throughout the code base. In the places where this variable is used, I don't think it's important to know that there's anything special about the variable. From the caller's perspective, it's just the version.